### DISCUSSION TITLE
Subfinder Giving double dashed and non valid urls

### DISCUSSION THREAD

**Discussion by hishammir** [projectdiscovery/subfinder#659] [Q&A]
**Describe the bug**
After running a huge scan , i noticed a lot of subdomains have `.-` in the subomains , some have `------` malformed URLs , that cannot be piped into other tools. Few of my API keys have exceeded the query quote, can that be a problem?

**Subfinder version**
Include the version of subfinder you are using, `subfinder -version`
`[INF] Current Version: v2.5.2`
**Complete command you used to reproduce this**
subfinder -dL domains.txt -o subs.txt

**Example of URLS**
```
ns1.-alpha.site.com
--in.comms.staging.site.com
-010140-mailman.site.ch
```

**Comment by ehsandeep** [comment]
@hishammir, thanks for opening the issue; to debug this further to identify the source, we will require the domain name producing result in the above format.
